src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
+				var dropdowntag = dropdown.tagName;
+				if (dropdowntag === 'DATALIST') {
+					var inputsibling = dropdown.previousElementSibling;
```

Comment:
You should move the variable declarations to the top of the function with the other variables. You can declare it like this:

```
var dropdowntag = dropdown.tagName;
var inputsibling = null;
etc.
```

And then use it inside the `if` statement:

```
if (dropdowntag === 'DATALIST') {
    inputsibling = dropdown.previousElementSibling;
etc.
```

Javascript "hoists" variables, and behaves as-if they were declared at the top of the function, so its clearer if you declare them there. See the [section on MDN about variable hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting) for more info.

It's also recommended to not declare variables inside an `if` statement. This [stackoverflow question](https://stackoverflow.com/questions/11532072/defining-javascript-variables-inside-if-statements) does an ok job of explaining it.